src/components/fetchData.jsx

```diff
@@ -28,6 +28,10 @@ export default function FetchData() {
     return <div className="error">Something went wrong!</div>;
   }
   return (
-    isLoading ? <div className="space">Loading... </div> : <Quotes text={quote.quote} author={quote.author} />
+    <div className="container div-center">
+      {
+        isLoading ? <div className="space">Loading... </div> : <Quotes text={quote.quote} author={quote.author} />
+      }
+    </div>
```

Comment:
- You have done great so far. :+1: :+1: Your UI looks like the provided layout. But there is an issue you need to fix before moving to the next milestone. 
- [ ] Currently, there is an issue in the `quotes.js`. The props validation gives an error in the console. I'll suggest you fix this as your app is required to run without errors. :+1: 

- - Screenshot:
![Selection_005](https://github.com/Nelofarzabi/math-magicians/assets/107632780/18420dbc-79e6-4c13-8d43-c814efcb0187)


You can fix this by updating your quote state in `FetchData` component to this.
```sh
  const [quote, setQuote] = useState({ quote: '', author: '' });

```